verilog/formatting/token_annotator_test.cc

```diff
@@ -4672,6 +4672,15 @@ TEST(TokenAnnotatorTest, AnnotateFormattingWithContextTest) {
           {/* any context */},
           {0, SpacingOptions::MustWrap},
       },
+      {
+          // return {<<8{data}};
+          DefaultStyle,
+          {verilog_tokentype::TK_return, "return"},
+          {'{', "{"},
+          {/* unspecified context */},
+          {NodeEnum::kStreamingConcatenation},
+          {1, SpacingOptions::Undecided},
+      },
```

Comment:
More suggested return test cases:

* `return (`
* `return $foo` (is a `SystemTFIdentifier`)
* `return -` (like `return -1`)
* `return !` (like `return !condition`)
* `return ~` (like `return ~condition`)

Reply:
Also mirror some of these tests over to formatter_test.cc, e.g.

```
{"function int f;return-1;endfunction",
 "function int f;\n"
 "  return -1;\n"
 "endfunction\n"},
{"function int f ;return    ! x\n;endfunction",
 "function int f;\n"
 "  return !x;\n"
 "endfunction\n"},

```